testapp/views.py

```diff
@@ -189,19 +169,21 @@ def query_api(term, location):
 
     businesses = response.get('businesses')
 
+    
     if not businesses:
         print(u'No businesses for {0} in {1} found.'.format(term, location))
         return
 
-    business_id = businesses[0]['id']
+    response_dic={}
+    for business in businesses:
+    	business_id=business['id']
+	#print(u'{0} businesses found, querying business info or the top result "{1}" ...'.format(len(businesses), business_id))
+    	test_result = get_business(bearer_token, business_id)
+    	print(u'Result for business "{0}" found:'.format(business_id))
+    	pprint.pprint(test_result, indent=2)
```

Comment:
I think if we're going print things out we should use the `logging` module (https://docs.python.org/3/library/logging.html).

You can make a new logging object using:

```
import logging

logging.basicConfig(level=logging.INFO)
logger = logging.getLogger('VIEWS')
```